### PR TITLE
Jormungandr: link fixes

### PR DESCRIPTION
**Pull request by TeXitoi**
fix http://jira.canaltp.fr/browse/NAVITIAII-2138

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/make_links.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def wrapper(*args, **kwargs):
                 data = self.prepare_objetcs(data)
                 kwargs = self.prepare_kwargs(kwargs, data)
                 for link in self.links:
-                    data["links"].append(create_external_link("v1.{link}".format(link=link), rel='related', templated=True, **kwargs))
+                    data["links"].append(create_external_link("v1.{link}".format(link=link), rel=link, templated=True, **kwargs))
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/make_links.py]: minor: length

**Review by pbougue — APPROVED** [review]

File: source/jormungandr/jormungandr/modules/v1_routing/resources/Index.py [hunk 2]
```diff
@@ -59,9 +59,9 @@ def get(self):
                                      rel='journeys',
                                      description='Compute journeys'),
                 # TODO: Activate for national autocomplete
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/modules/v1_routing/resources/Index.py]: you can remove the comment above

File: source/jormungandr/jormungandr/modules/v1_routing/resources/Index.py [hunk 3]
```diff
@@ -59,9 +59,9 @@ def get(self):
...
-                # create_external_link(self.module_name + '.places',
-                #                      rel='places',
-                #                     description='Autocomplete api'),
+                create_external_link(self.module_name + '.places',
+                                     rel='places',
+                                     description='Autocomplete api'),
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/modules/v1_routing/resources/Index.py:63]: :+1:

File: source/jormungandr/tests/tests_mechanism.py [hunk 4]
```diff
@@ -266,6 +266,18 @@ def check_journeys_links(self, response, query_dict):
 
                 eq_(query_dict[k], v)
 
+        if has_pt and '/coverage/' in query:
+            types = [l['type'] for l in response['links']]
+            print(types)
```
Comment (antoine-de) [inline source/jormungandr/tests/tests_mechanism.py]: debug line

**Review by antoine-de — COMMENTED** [review]
great new tests!

**Review by antoine-de — APPROVED** [review]

**Pull request merged**